src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/OffenderEventsAdminService.kt

```diff
@@ -20,6 +20,15 @@ class OffenderEventsAdminService(
   @Value("\${offender-events-sqs.dlq.name}") private val dlqName: String
 ) : QueueAdminService(awsSqsClient, awsSqsDlqClient, queueName, dlqName)
 
+@Service
```

Comment:
Not that important but the file name ought to be changed if it has all the queue's in

Reply:
Will update in next PR